.storybook/components/Docs/Guildelines/Tokens.stories.mdx

```diff
@@ -84,133 +87,88 @@ Tier 1 tokens define the brand’s design values. They can be thought of as a br
 
-- `primary-foreground` defines a branded text color
-- `primary-foreground-hover` defines a branded text hover color
-- `primary-background` defines a branded background color
-- `primary-background-hover` defines a branded background hover color
-- `primary-border` defines a branded border
-- `primary-border-hover` defines a branded hover border
-- `primary-foreground-inverted` defines a branded text color rendered on a dark background
+Each token in this group is prefixed by `--eds-theme-color` and has certain suffixes based on the the broad categorization.
 
-Additional branded values ("secondary", "tertiary", "accent", "supports", etc) can be added to accomplish desired UI results.
+For more information, [refer to Storybook](/story/design-tokens-tier-2-usage--colors) for the existing tokens and their naming.
 
-#### Disabled colors
-
-- `disabled-foreground` defines a disabled text color
-- `disabled-background` defines a disabled background color
-- `disabled-border` defines a disabled border
-
-#### Focus colors
-
-- `focus-ring` defines the focus ring color used for component `:focus` state
-
-#### Link colors
+### Forms <a name="tier-2-forms"></a>
 
-- `link-foreground` defines text link color. By default, this is mapped to `eds-theme-color-primary-foreground`.
-- `link-foreground-hover` defines text link hover color
+Defines the border width, radius, and color properties for form controls. These values are all mapped to other Tier 2 values and are considered optional in case remapping is required.
 
-#### Neutral colors
+- `form-border-width` defines the default border width for any form element
+- `form-border-radius` defines the default border radius used on any form element's border
 
-- `neutral-subtle` defines a subtle border
-- `neutral-medium` defines a medium border
-- `neutral-bold` defines a pronounced border
+## Tier 3 Component Tokens <a name="tier-3-component-tokens"></a>
 
-#### Text highlight colors
+Tier 3 component variables represent each component's specific values.
 
-Defines text highlight values
+Naming for tier-3 tokens will use the following prefix:
 
-### Forms <a name="tier-2-forms"></a>
+```css
+--eds-theme-[property]-[component name]
+```
 
-Defines the border width, radius, and color properties for form controls. These values are all mapped to other Tier 2 values and are considered optional in case remapping is required.
+With the rest matching a corresponding use for the given component. For example, if a modal is to have a specific header background color, the token to use is:
 
-### Shadows <a name="tier-2-shadows"></a>
+```css
+--eds-theme-color-brand-header-background
+```
 
-- `box-shadow` defines the shadow value that gets applied by default to buttons and other "elevated" components like cards.
+The suffix here is arbitrary, but uses similar naming to tier two tokens. When possible, any new tier 3 tokens should be a more specifically named tier 2 token, to keep the naming consistent.
```

Comment:
@jinlee93 to make sure this section matches what you have for naming in #1623 ... I just realized my example leaves out `-modal` lol. Otherwise, does this seem accurate?